app/src/main/java/org/dhis2/usescases/main/HomeNavigator.kt

```diff
@@ -23,7 +23,7 @@ sealed class HomeItemData(
         override val label: String,
         override val accessDataWrite: Boolean,
         val trackedEntityType: String,
-        val stockConfig: AppConfig?,
+        val stockConfig: StockUseCase?,
```

Comment:
Maybe we don't need the object, given that is only used to choose navigate to the stock use case or the regular search. What do you think about changing it by a `isStockUseCase: Boolean` ?